src/lib/git.sh

```diff
@@ -0,0 +1,51 @@
+# The git.sh file provides git easy functionalities for kworkflow code.
+
+# Run git bisect
+#
+# @good: good commit hash
+# @bad: bad commit hash
+# @command: test performed by git bisect
+function run_git_bisect()
+{
+  local good="$1"
+  local bad="$2"
+  local command="$3"
+
+  git bisect start
+  if [[ $? != 0 ]]; then
```

Comment:
Add double quotes around `$?`. Also, it is worth exploring if git bisect has different return errors that we can handle here.